src/fparser/tests/test_issue11.py

```diff
@@ -67,7 +67,7 @@
 def test_reproduce_issue():
     source_str = '''\
       subroutine bndfp()
-      include "events.ins"
+      ! include "events.ins"
```

Comment:
Can this test just be removed altogether now?

Reply:
It is failing because the reader now passes includes onto fparser1 which does not recognise the syntax. I've just made it xfail as that seemed to be the correct thing.

Reply:
Great. That's nicer.